pkgs/development/python-modules/mesa/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, isPy27
+, cookiecutter, networkx , pandas, tornado, tqdm
+, pytest }:
+
+buildPythonPackage rec {
+  pname = "mesa";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    repo = "mesa";
+    owner = "projectmesa";
+    rev = "v${version}";
+    sha256 = "0d8c636zhswxd91ldlmdxxlyym2fj3bk1iqmpc1jp3hg7vvc7w03";
+  };
+
+  # According to their docs, this library is for Python 3+.
+  disabled = isPy27;
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
```

we should comment as to why this is disabled
```suggestion
  # ignore test which tries to mkdir in unreachable location
  checkPhase = ''
```

---

Yes, definitely.